docs-source/content/userguide/managing-domains/model-in-image/overview.md

```diff
@@ -27,7 +27,7 @@ It enables:
  - Embedding model files and archives in a custom Docker image, and using the WebLogic Image Tool (WIT) to generate this image.
  - Supplying additional model files using a Kubernetes ConfigMap.
  - Supplying Kubernetes Secrets that resolve macro references within the models. For example, a secret can be used to supply a database credential.
- - Updating WDT model files at runtime. For example, you can add a data source to a running domain. Note that all such updates currently cause the domain to 'roll' in order to take effect.
+ - Updating WDT model files at runtime. For example, you can add a data source to a running domain. See [Runtime updates](#runtime-updates) for details.
```

Comment:
- The runtime-updates section of this document needs to be updated to briefly discuss offline vs. online, and their respective rolling behavior - we can word-smith that together if you like.
- Does debugging document need to be updated to with advice that'd be helpful for debugging online updates?  (For example, which conditions/events/severes to look for...)